### ISSUE TITLE
Do we really need to provide an argument for -s option?

### ISSUE THREAD

**Issue by c02y** [noctuid/tdrop#77]
When I try to run `tdrop -ma -w 100% -h 95% -y "$PANEL_HEIGHT" -s  kitty`, it fails and says:
```
Error: Program to run is required as a positional argument. For help use -h or --help or see the manpage.
```
So in the tdrop script, `-s` need an option for `session_name`, but what if I don't want a session name manually, normally I just use `tmux`, I got my own `~/.tmux.conf` and set `status-left` as `#S`, the `#S` is the session-name, I never provide a session name manually, so by default, this value will be an increasing number start from 0.

So I suggest the argument for `-s` should be optional, if no argument provided, use the tmux default, it provided, use the provided one.

**UPDATE**:
Actually I personally don't need `-s` option, I start a script call `t` which is a wrapper for tmux in kitty.conf, so this is actually what I need.

But the original suggestion still needs to be done.

**Comment by noctuid** [comment]
If you want to do this, you can just run whatever command you want to in the terminal instead of using `-s` (e.g. `tdrop kitty tmux`).

**Comment by c02y** [comment]
Yes, I got it now. After search other people's usage in the whole github, I managed to make my needs work.

But I think it is better to provide something like a wiki page for some examples for regular programs.

**Comment by c02y** [comment]
Wait... you mean "tdrop kitty tmux", the line in my page got wrapped after the word `kitty`, I didn't notice that I need to use tmux at the end.

I tried `tdrop kitty tmux` in terminal manually, it starts kitty with empty content, without even a shell prompt, and I cannot do typing in it.

Actually I managed to use tdrop + kitty using `tdrop -ma -w 100% -h 90% kitty`, and I use a shell script which is a wrapper of tmux as the shell command in kitty.conf, I usually use this shell script as the start command in other terminals such as xfce4-terminal or yakuake.
